terraform/modules/private-dns-zone-extend/main.tf

```diff
@@ -0,0 +1,34 @@
+provider "aws" {
+  alias = "core-network-services"
+  version = "~> 2.0"
+}
+
+resource "aws_route53_vpc_association_authorization" "private_zone_vpc_auth" {
+provider = aws.core-network-services
+vpc_id   = var.vpc_id
+zone_id  = var.business_unit_name
+}
+
+resource "aws_route53_zone_association" "private_zone_assoc"  {
```

Comment:
This looks fine, but I'd harmonise the names (EG. `private_zone_*` or `private_zone_vpc_*`)

Reply:
changed the name so that they match